robottelo/ui/base.py

```diff
@@ -209,6 +212,25 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
         try:
             element = WebDriverWait(
                 self.browser, timeout, poll_frequency
+            ).until(expected_conditions.presence_of_element_located(locator))
+            self.wait_for_ajax(poll_frequency=poll_frequency)
```

Comment:
If ```WebDriverWait().until()``` return ```None``` and we have infinite Ajax procedure running in background(worst scenario) we will loose 12 seconds for our run, but we use ```wait_for_ajax``` for another purpose for situation when waiter above will return ```not None``` value. I have faced next scenario a lot on my previous project and believe that it can be actual for current project too.